packages/api/test/codegen/languages/typescript/index.test.ts

```diff
@@ -124,6 +140,14 @@ describe('typescript', () => {
   });
 
   describe('#compile', () => {
+    beforeEach(() => {
+      vi.setSystemTime(new Date('2023'));
```

Comment:
Was about to ask to do some sort of `<<package version>>` thing for the date in the README then noticed this.

![pretty-sneaky-bro-sneaky](https://github.com/readmeio/api/assets/33762/cb7f547c-403a-4593-9de0-af69cab17642)